_slides/02-vvaths.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
```

Comment:
Looks great, please make the requested change

Reply:
done.